aws2wrap/__init__.py

```diff
@@ -328,7 +364,8 @@ def main():
         print(json.dumps(output))
     else:
         run_command(access_key, secret_access_key, session_token, profile, args)
+    return 0
 
 
 if __name__ == '__main__':
-    main()
+    sys.exit(main())
```

Comment:
Can you please explain/clarify this change? As far as I can tell, line 367 means that main will *always* return 0. Python implicitly does a `sys.exit(0)` if the code finishes normally, so I'm trying to understand the purpose of lines 367 and 371.

Reply:
That's more of something I always do with my own scripts. Ideally there should be only one sys.exit() in the entire script, with everything else throwing exceptions with a `try/except` block in `main()`. This allows for cleaner unittesting of internal functions without having to mock `sys.exit()` itself, but also as others might import these functions they might not want to have these functions `sys.exit()` and might want to more gracefully handle the error.

I did not want to overload this PR with too many changes but I'm ok with reverting this change and maybe to the `sys.exit()` cleanup in a separate PR.

Reply:
@sodul Thanks for the clarification. There is an extra `sys.exit` in `main()` so it might be cleaner to deal with the `sys.exit()` cleanup separately.

Reply:
done